tests/test_random.py

```diff
@@ -1683,6 +1683,40 @@ def test_ne():
     assert repr(d1) == repr(d2)
     assert d1 != d2
 
+@timer
+def test_int64():
+    # cf. #1009
+    # Check that various possible integer types work as seeds.
+
+    rng1 = galsim.BaseDeviate(int(123))
+    # cf. https://www.numpy.org/devdocs/user/basics.types.html
+    ivalues =[np.int8(123),  # Note this one requires i < 128
+              np.int16(123),
+              np.int32(123),
+              np.int64(123),
+              np.uint8(123),
+              np.uint16(123),
+              np.uint32(123),
+              np.uint64(123),
+              np.short(123),
+              np.ushort(123),
+              np.intc(123),
+              np.uintc(123),
+              np.intp(123),
+              np.uintp(123),
+              np.int_(123),
+              np.longlong(123),
```

Comment:
I suppose `np.long` and `np.ulong` should be added for completeness.

Reply:
I think I got all the numpy valid types.  It's true that 
`np.long` is valid (albeit [undocumented](https://docs.scipy.org/doc/numpy/user/basics.types.html)).  But it is equivalent to `int` in Python 3.x and `long` in Python 2.7, so not really an additional type to check.  

However, I don't think `np.ulong` is valid.
```
>>> np.long
<class 'int'>
>>> np.ulong
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'numpy' has no attribute 'ulong'
```

Reply:
Oh interesting.  I think I somehow ended up at the [v1.10 documentation](https://docs.scipy.org/doc/numpy-1.10.1/user/basics.types.html) which implies that long and ulong are a thing.  But I guess this doesn't exist in recent numpy.